design/one-pager-stack-status-spec.md

```diff
@@ -0,0 +1,82 @@
+# Stack UI Schema Status Spec
+
+- Owner: Steven Rathbauer (@rathpc)
+- Reviewers: Crossplane Maintainers
+- Status: Draft
+
+## Proposal
+
+[crossplaneio/crossplane#929](https://github.com/crossplaneio/crossplane/issues/929)
+
+## Problem
+
+As of right now there is no defined spec to be able to drive status information for a given stack or resource from a GUI perspective. It would be useful to add keys to the existing `ui-schema.yaml` file that can be used for this purpose.
```

Comment:
What would you think of wrapping the long lines earlier? It makes it easier to comment on the lines in github since the lines end up shorter, and they show up the same in the rendered markdown : )

Reply:
I can do that!